games/superopti.py

```diff
@@ -71,6 +75,9 @@ def Draw(self):
             return None
 
         game_img = self.framebuffer
+        if self.scalefactor != 1:
+            new_size = tuple([size * self.scalefactor for size in self.framebuffer.get_size()])
```

Comment:
you can actually drop the square brackets here, tuple() can take an iterable that need not be a list